src/plugins/intel_cpu/src/nodes/convert.cpp

```diff
@@ -123,13 +131,33 @@ void Convert::initSupportedPrimitiveDescriptors() {
             config.inConfs[0].setMemDesc(std::make_shared<CpuBlockedMemoryDesc>(itr->second->createDesc(insPrecision, insShape)));
             config.outConfs[0].setMemDesc(std::make_shared<CpuBlockedMemoryDesc>(itr->second->createDesc(outPrecision, outputShape)));
 
-            supportedPrimitiveDescriptors.emplace_back(config, impl_desc_type::unknown);
+            supportedPrimitiveDescriptorsBuilder(config);
         }
     } else {
         IE_THROW() << errorPrefix << " has incorrect number of input/output edges";
     }
 }
 
+void Convert::prepareParams() {
+    auto& parentMem = getParentEdgeAt(0)->getMemory();
+    convertParams.size = parentMem.GetDescWithType<BlockedMemoryDesc>()->getPaddedElementsCount();
+
+    auto selectedPD = getSelectedPrimitiveDescriptor();
+    std::vector<MemoryDescPtr> srcDescs;
+    for (size_t i = 0; i < getOriginalInputsNumber(); i++) {
+        srcDescs.push_back(getParentEdgeAt(i)->getMemoryPtr()->getDescPtr());
+    }
+    std::vector<MemoryDescPtr> dstDescs;
+    for (size_t i = 0; i < getOriginalOutputsNumber(); i++) {
+        dstDescs.push_back(getChildEdgeAt(i)->getMemoryPtr()->getDescPtr());
+    }
```

Comment:
`OriginalInputsNumber` and `riginalOutputsNumber` doesn't necessarily correspond to the actual number of input and output ports of the CPU node. It's more safe to use port configs. However, given the semantics of the `Convert` operation, does it have more than one input and one output ports? If so, does it make sense to use vectors then?